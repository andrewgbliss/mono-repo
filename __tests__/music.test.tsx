import { render } from '@testing-library/react';
import Music from '../pages/music';

it('renders music unchanged', () => {
  const { container } = render(<Music />);
  expect(container).toMatchSnapshot();
});
