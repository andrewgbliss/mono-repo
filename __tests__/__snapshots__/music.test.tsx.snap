// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders music unchanged 1`] = `
<div>
  <div
    class="font-mono flex flex-wrap flex-col h-screen"
  >
    <div
      class="flex-grow bg-[url('/img/andrewguitar.jpg')] bg-cover sm:bg-auto bg-black bg-center bg-repeat flex justify-center items-center pb-12"
    >
      <div
        class="shadow-2xl rounded-lg p-2 p-2"
      >
        <h1
          class="text-center p-2 text-3xl text-white"
        >
          Latest Album
        </h1>
        <div
          class="flex justify-center gap-5 p-5"
        >
          <div
            class="bg-white p-2 rounded-xl"
          >
            <ol>
              <li
                class="bg-black text-white cursor-pointer"
              >
                Intro
              </li>
              <li
                class="cursor-pointer hover:bg-black hover:text-white"
              >
                Just Go
              </li>
              <li
                class="cursor-pointer hover:bg-black hover:text-white"
              >
                The Most Important Thing
              </li>
              <li
                class="cursor-pointer hover:bg-black hover:text-white"
              >
                To A Tragedy
              </li>
              <li
                class="cursor-pointer hover:bg-black hover:text-white"
              >
                Alright
              </li>
              <li
                class="cursor-pointer hover:bg-black hover:text-white"
              >
                Red Tied Blood
              </li>
              <li
                class="cursor-pointer hover:bg-black hover:text-white"
              >
                Fight Back
              </li>
              <li
                class="cursor-pointer hover:bg-black hover:text-white"
              >
                Two Face
              </li>
              <li
                class="cursor-pointer hover:bg-black hover:text-white"
              >
                Push
              </li>
              <li
                class="cursor-pointer hover:bg-black hover:text-white"
              >
                Wasteland
              </li>
              <li
                class="cursor-pointer hover:bg-black hover:text-white"
              >
                Line Of
              </li>
              <li
                class="cursor-pointer hover:bg-black hover:text-white"
              >
                Destruction
              </li>
            </ol>
          </div>
          <div
            class="bg-black flex-col justify-center hidden sm:flex rounded-xl"
          >
            <span
              style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
            >
              <span
                style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
              >
                <img
                  alt=""
                  aria-hidden="true"
                  src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27312%27%20height=%27256%27/%3e"
                  style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                />
              </span>
              <img
                alt=""
                data-nimg="intrinsic"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
              />
              <noscript />
            </span>
          </div>
        </div>
        <div
          class="flex justify-center p-2"
        >
          <audio
            controls=""
            id="audio"
          >
            <source
              id="audioSource"
              src="/audio/intro.mp3"
              type="audio/mpeg"
            />
            Your browser does not support the audio element.
          </audio>
        </div>
      </div>
    </div>
  </div>
</div>
`;
